config.go

```diff
@@ -49,6 +50,9 @@ const (
 	defaultMaxLogFiles        = 3
 	defaultMaxLogFileSize     = 10
 
+	defaultTorDNS       = "soa.nodes.lightning.directory:53"
```

Comment:
This is the DNS server that we connect to in order to query for the SRV records needed, so this can be any DNS server I believe. The chain specific DNS server is already handled at the server level [here](https://github.com/lightningnetwork/lnd/blob/master/server.go#L634).